packages/circuit-ui/components/RadioButton/RadioButton.tsx

```diff
@@ -36,6 +36,10 @@ export interface RadioButtonProps
    * A clear and concise description of the option's purpose.
    */
   label: string;
+  /**
+   * Further details about the option's purpose.
+   */
+  helperText?: string;
```

Comment:
The [Toggle](https://circuit.sumup.com/?path=/docs/forms-toggle--docs) and [Selector](https://circuit.sumup.com/?path=/docs/forms-selectorgroup--docs) components have a similar prop called `description`. Let's keep the naming consistent.

```suggestion
  description?: string;
```